src/common/modal/modal.module.scss

```diff
@@ -24,4 +24,8 @@
   &:focus {
     outline: none;
   }
+  width: 100%;
```

Comment:
Style has no effect

Reply:
Actually it does, this makes the modal grow to the given `max-width`, and also makes it scale down on smaller screen sizes (though our responsive design is still practically non-existent)

Reply:
The react modal is rendered as block level element (`div`), so it naturally takes the whole width of the parent container. There is no need to specify `width: 100%` for a block level elements. Do you agree? :)

Reply:
Ah, indeed. It also looks to break modals without a defined size, so it makes sense to remove it.